arches_vue_utils/src/arches_vue_utils/utils.ts

```diff
@@ -0,0 +1,64 @@
+import { ALT_LABEL, PREF_LABEL } from "@/arches_vue_utils/constants.ts";
+
+import type {
+    HasLabels,
+    HasValues,
+    Label,
+    Labellable,
+} from "@/arches_vue_utils/types";
+
+/* Port of rank_label in arches.app.utils.i18n python module */
+export const rankLabel = (
```

Comment:
non-blocking nit: I've started developing a pretty strong opinion about function declarations vs function expressions. It really doesn't matter in this context but if you've also got strong opinions in general it'd be fun to chat 😄